classes/ZipAction.php

```diff
@@ -151,11 +151,19 @@ public function extract($from_file, $to_dir)
             return false;
         }
 
-        if (!$zip->extractTo($to_dir)) {
-            $zip->close();
-            $this->logger->error($this->translator->trans('zip->extractTo(): unable to use %s as extract destination.', array($to_dir), 'Modules.Autoupgrade.Admin'));
+        for($i = 0; $i < $zip->numFiles; $i++) {
```

Comment:
```suggestion
        for ($i = 0; $i < $zip->numFiles; $i++) {
```

Reply:
Will be automatically fixed with #303. I won't fix in this issue to avoid triggering Travis